futures-util/src/future/abortable.rs

```diff
@@ -121,16 +122,24 @@ pub fn abortable<Fut>(future: Fut) -> (Abortable<Fut>, AbortHandle)
 }
 
 /// Indicator that the `Abortable` future was aborted.
-#[derive(Copy, Clone, Debug, Eq, PartialEq)]
```

Comment:
Same comment as with `Canceled`-- this is nice for matching on, and I can't forsee wanting to add anything here.